### PR TITLE
dev-db/mycli: add 1.27.0

### PR DESCRIPTION
**Pull request by parona-source**

### CODE REVIEW

File: dev-db/mycli/mycli-1.27.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+)
+
+EPYTEST_IGNORE=(
+	# Requires a running mysql daemon
+	"test/test_sqlexecute.py"
+)
+
+distutils_enable_tests pytest
+
+PATCHES=( "${FILESDIR}/mycli-1.21.1-fix-test-install.patch" )
```
Comment (mgorny) [inline dev-db/mycli/mycli-1.27.0.ebuild:74]: Is there a PR for this? It looks really bad that we need to carry this patch for that many versions.

Comment (parona-source) [reply dev-db/mycli/mycli-1.27.0.ebuild:74]: For what I can see no. I'll have a proper look at this, as this was in the ebuild before I took over so I took it as a "if it aint broke dont fix it".

Comment (parona-source) [reply dev-db/mycli/mycli-1.27.0.ebuild:74]: Seeing that it installs the test package even with regular "pip install mycli" from the wheel. Id think upstream should be interested in this.

Comment (mgorny) [reply dev-db/mycli/mycli-1.27.0.ebuild:74]: Precisely. It is broken. BTW easiest to test by doing `python -m build -w`.

Comment (parona-source) [reply dev-db/mycli/mycli-1.27.0.ebuild:74]: https://github.com/dbcli/mycli/pull/1126

**Pull request closed without merging**